website/blog/2025-01-28-git-branching-strategies-and-dbt.md

```diff
@@ -0,0 +1,611 @@
+out simply\:
+
+- What is the strategy?
+- How does the development workflow of the strategy look to a team?
+- Which **repository branching rules and helpers** help us in this strategy?
+- How do we commonly configure **dbt Cloud** for this strategy?
+- How do branches and dbt processes map to our **data platform** with this strategy?
+
+Then, we’ll end by comparing the strategies and covering some frequently asked questions.
+
+:::info[Know before you go]
+
+There are *many* ways to configure each tool (especially dbt) to accomplish what you need. The upcoming 
+strategy details were written intently to provide what we think are the minimal standards 
+to get teams up and running quickly. These are starter configurations and practices which 
+are easy to tweak and adjust later on. Expanding on these configurations is
+an exercise left to the reader!
+
+:::
+
+## Direct promotion
+
+**Direct promotion** means we only keep one long-lived branch 
+in our repository — in our case, `main`. Here’s the workflow for this strategy\:
+
+<Lightbox src="/img/blog/2025-01-28-git-branching-strategies-and-dbt/2_direct_git.png" title="Direct promotion branching strategy" width="85%" />
+
+### How does the development workflow look to a team?
+
+Layout\:
+
+- `feature` is the developer’s unique branch where task-related changes happen
+- `main` is the branch that contains our “production” version of code
+
+Workflow\:
+
+- **Development**\: I create a `feature` branch from `main` to make, test, and personally review changes
+- **Quality Assurance**\: I open a pull request comparing my `feature` against `main`, which is then reviewed by peers (required), stakeholders, or subject matter experts (SMEs). We highly recommend including stakeholders or SMEs for feedback during PR in this strategy because the next step changes `main`.
+- **Promotion**\: After all required approvals and checks, I merge my changes to `main`
+- **Deployment**\: Others can see and use my changes in `main` after I merge and `main` is deployed
+
+### Repository branching rules and helpers
+At a minimum, we like to set up\:
+- **Branch protection** on `main` ([like these settings for GitHub](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches)), requiring\:
+  - a pull request (no direct commits to `main`) 
+  - pull requests must have at least 1 reviewer's approval
+- **A PR template** ([such as our boiler-plate PR template](https://docs.getdbt.com/blog/analytics-pull-request-template)) for `feature` PRs against `main`
+
+### dbt Cloud processes and environments
+
+Here’s our branching strategy again, but now with the dbt Cloud processes we want to incorporate\:
+
+<Lightbox src="img/blog/2025-01-28-git-branching-strategies-and-dbt/3_direct_dbt_deployment.png" title="Direct Promotion strategy with dbt cloud processes denoted" width="85%" />
```

Comment:
@christineberger this one needs a forward slash in front of `img`

Reply:
And same for the rest of the images below in this file. It looks like they just need that forward slash on the image src and they should be good to go!